ray-operator/config/samples/ray-service.mobilenet.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: ray.io/v1alpha1
+kind: RayService
+metadata:
+  name: rayservice-mobilenet
+spec:
+  serviceUnhealthySecondThreshold: 300 # Config for the health check threshold for service. Default value is 60.
+  deploymentUnhealthySecondThreshold: 300 # Config for the health check threshold for deployments. Default value is 60.
+  serveConfig:
+    importPath: mobilenet.graph
+    runtimeEnv: |
+      working_dir: "https://github.com/kevin85421/ray-serve-examples/archive/main.zip"
+      pip: ["python-multipart"]
```

Comment:
Could you pin this to the current version? That prevents any regressions from future versions affecting this example.

Reply:
Resolved by https://github.com/ray-project/kuberay/pull/1175/commits/feafb3b99cf929477d2124d456dadfd36cc62def. I used the following instructions to obtain the version of `python-multipart`. Since I'm not familiar with the runtime environment, I'm unsure if this is the correct method.

```sh
cd /tmp/ray/session_latest/runtime_resources/pip/a8c698daaafc716fd7f7e828c91158e9610b9424/virtualenv/bin
./pip freeze | grep multipart
```
<img width="1353" alt="Screen Shot 2023-06-20 at 10 03 05 PM" src="https://github.com/ray-project/kuberay/assets/20109646/bdd0780f-a8ad-4bab-8c4f-cdbad4897b90">

Reply:
I will resolve this comment after I confirm with Archit tomorrow.

Reply:
Hi Archit, does https://github.com/ray-project/kuberay/pull/1175#discussion_r1236284166 make sense to you? Thank you!

Reply:
Yeah, that's the right syntax for pinning the version in the runtime env.  We just pass it directly to `pip install`